### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Bestbynature**
## Features included in this pull request are as follows: 

- [x] A new React application was set up using `Create React App`
- [x] A directory for my reusable `/components` components was created 
- [x] React Router V6 was installed and set up

- [x] 2 routes were added in the  application, each rendering a page component:
-  /
- /categories
- [x] Components were created inside the `/components` directory that:
- [x] Contains individual book states, as well as a button to delete the book
- [x] Contains a list to render individual books
- [x] Contains a form to create new books
- [x] Contains navigation link to the 2 routes earlier created
- [x] The input for the category has been replaced with the author as required in this milestone.

### CODE REVIEW

File: src/components/App.js
Comment (emmiiorji) [inline src/components/App.js:1]: You seem to have altered the folder structure created by the [Create React App](https://create-react-app.dev/)using [Create React App](https://create-react-app.dev/). This file is usually directly in the src folder. Kindly adjust move it to be there.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Bestbynature, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Routes are working as expected
- Good job splitting into several components

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/App.js
Comment (Bestbynature) [reply src/components/App.js:1]: Thank you so much. I appreciate your careful observation of my work.

**Review by danmainah — APPROVED** [review]
Hi @Bestbynature , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**